client/src/components/Account/EditProfileForm/EditProfileForm.tsx

```diff
@@ -1,52 +1,140 @@
+            label="FIRST NAME"
+            placeholder="First Name"
+          />
+          <CustomTextField
+            onChange={(e: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLTextAreaElement>) =>
+              handleChange(e, 'lastName')
+            }
+            value={profile.lastName ? profile.lastName : ''}
+            label="LAST NAME"
+            placeholder="Last Name"
+          />
+          <CustomTextField
+            onChange={(e: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLTextAreaElement>) =>
+              handleChange(e, 'email')
+            }
+            value={profile.email ? profile.email : ''}
+            label="EMAIL ADDRESS"
+            placeholder="user@gmail.com"
+          />
           <Grid item xs={12}>
             <Grid container alignItems="center" spacing={2}>
               <Grid item xs={12} sm={3}>
                 <Typography variant="body1" align="right" className={classes.formLabel}>
                   PHONE NUMBER
                 </Typography>
               </Grid>
-              <Grid item xs={12} sm={4}>
+              <Grid item xs={12} sm={4} className={`${showPhoneInput && classes.shouldNotDisplay}`}>
                 <Typography align="left" variant="body1" className={classes.phoneNumber}>
                   No Phone number entered
                 </Typography>
               </Grid>
-              <Grid item xs={12} sm={4}>
-                <Button color="primary" variant="outlined" size="large">
+              <Grid item xs={12} sm={4} className={`${showPhoneInput && classes.shouldNotDisplay}`}>
+                <Button color="primary" variant="outlined" size="large" onClick={toggleInput}>
                   Add a phone number
                 </Button>
               </Grid>
+              <Grid item xs={12} sm={8} className={`${!showPhoneInput && classes.shouldNotDisplay}`}>
+                <Grow in={showPhoneInput}>
+                  <TextField
+                    label="Phone Number"
+                    value={profile.phoneNumber ? profile.phoneNumber : ''}
+                    color="secondary"
+                    variant="outlined"
+                    fullWidth
+                    onChange={(e: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLTextAreaElement>) =>
+                      handleChange(e, 'phoneNumber')
+                    }
+                  />
+                </Grow>
+              </Grid>
             </Grid>
           </Grid>
-          <CustomTextField onChange={handleChange} value="" label="WHERE YOU LIVE" placeholder="Address" />
           <CustomTextField
-            onChange={handleChange}
-            value=""
+            onChange={(e: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLTextAreaElement>) =>
+              handleChange(e, 'address')
+            }
+            value={profile.address ? profile.address : ''}
+            label="WHERE YOU LIVE"
```

Comment:
For text you want to show as uppercase it's usually better to write it in normal case and then use [text-transform:uppercase](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform), otherwise some screen-readers treat it as an acronym.

Reply:
Ditto for fields above

Reply:
updated